hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/RowProjector.java

```diff
@@ -39,7 +39,7 @@
     private final ExpressionEvalContext evalContext;
 
     @SuppressWarnings("unchecked")
-    RowProjector(
+    public RowProjector(
```

Comment:
You can access it via `SqlProcessors`. It's not enough?

Reply:
Unfortunately, not!

Reply:
Ok, ok...